app/controllers/curated_clusters_controller.rb

```diff
@@ -19,12 +19,6 @@ def create
     redirect_to "/users/#{@user.username}/curated_clusters"
   end
 
-  def addarticle
```

Comment:
another comment I made that has already been incorporated. erik you rockstar

Reply:
Completed!